packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts

```diff
@@ -97,6 +97,12 @@ const styles = css`
   .select-selected > .close-icon {
     display: flex;
     align-items: center;
+    fill: var(--affine-icon-color);
```

Comment:
In our design system, the icon color here should be:

```suggestion
    fill: var(--affine-black-90);
```

Reply:
Thanks.